functions/controllers/message/sendMessages.js

```diff
@@ -0,0 +1,73 @@
+const HttpStatus = require('http-status-codes');
+const { status, message } = require('../../util/constants');
+
+const { error, success } = status;
+const { somethingWentWrong, messageSent } = message;
+const { INTERNAL_SERVER_ERROR, OK } = HttpStatus;
+
+const errors = {
+    error2: res => {
+        res.status(INTERNAL_SERVER_ERROR).json({
+            message: somethingWentWrong,
+            status: error,
+        });
+    },
+};
+
+const successMessage = {
+    success1: res => {
+        res.status(OK).json({
+            message: messageSent,
+            status: success,
+        });
+    },
+};
```

Comment:
I see that you have repeated this several places.
Try to have a global error message and success message functions and accept custom message as the case maybe.
```
const errors = {
    error2: (res, errorType, message) => {
        res.status(errorType).json({
            message,
            status: error,
        });
    },
};

const successMessage = {
    success1: (res, message) => {
        res.status(OK).json({
            message,
            status: success,
        });
    },
};
```
this way you can reuse it anywhere in your code